pkg/asset/machines/aws/awsmachines.go

```diff
@@ -33,6 +33,8 @@ type MachineInput struct {
 	PublicIP       bool
 	PublicIpv4Pool string
 	Ignition       *capa.Ignition
+	HostAffinity   string
+	HostIDs        []string
```

Comment:
These fields are not being used or set, right?

They need to be set in `pkg/asset/machines/clusterapi.go` and pass down to `AWSMachine` manifest here?